rancid-tomatillos/src/api.js

```diff
@@ -0,0 +1,36 @@
+const api = {
+  async getIndividualMovie(id, props) {
+    let promise = await fetch(
+      `https://rancid-tomatillos.herokuapp.com/api/v2/movies/${id}`
+    );
+    let movie = await promise.json();
+    props.history.push({
+      pathname: "/movie",
+      state: {
+        movie: movie.movie,
+      },
+    });
+  },
```

Comment:
I think we missed the error handling on `getIndividualMovie()` lets add that in next